CONTRIBUTING.md

```diff
@@ -51,6 +50,15 @@ For Javascript please make sure your code is formatted using the JS Beautify sta
 For Jade/PUG, you can use the following atom package: https://atom.io/packages/jade-beautify
 For HTML class names, please use all lower case name, with tokens separated by dashes (e.g. edit-blog-button). For HTML ID's, please use on alphabetic characters in camelcase.
 
+### Development Server
+
+The development server can be accessed at: http://162.243.163.51/
+This server runs off the master branch, uses http instead of https, and has it's own database. The development server is a great way to test new features in an environment closer to the live server. Please report any bugs you see on the development server as an issue.
```

Comment:
Is the dev server database wiped periodically?

Reply:
It is not, but it is subject to it. I'll add that comment.